### DISCUSSION TITLE
Cancel eating food

### DISCUSSION THREAD

**Discussion by DevBlokJochem** [Minestom/Minestom#1270] [Help]
Hello, how can I stop players from eating food? (and the animation)
I have tried this:
```
private fun fixGapple(event: PlayerPreEatEvent) {
        event.isCancelled = true
    }

    private fun fixGapple(event: PlayerHandAnimationEvent) {
        event.isCancelled = true
    }
```
but this doesnt work. the player is still eating the golden apple.

**Comment by asyncrom** [comment]
`event.setCancelled(true);` ?

**Reply by DevBlokJochem** [reply]
Hey,
thanks for trying to help, but sadly this doesnt work. Im coding in kotlin, and it is the same code as you wrote.
